app/src/main/java/com/framgia/soundcloud/screen/home/offline/OfflineViewModel.java

```diff
@@ -1,16 +1,37 @@
 package com.framgia.soundcloud.screen.home.offline;
 
+import android.content.ContentResolver;
+import android.content.ContentUris;
```

Comment:
ERROR: Unused import: android.content.ContentUris. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports